core/src/main/java/hudson/init/TaskMethodFinder.java

```diff
@@ -116,8 +116,8 @@ protected void invoke(Method e) {
      */
     private Object lookUp(Class<?> type) {
         if (type==Jenkins.class || type==Hudson.class)
-            return Jenkins.getInstanceOrNull();
-        Jenkins j = Jenkins.getInstanceOrNull();
+            return Jenkins.getInstance();
+        Jenkins j = Jenkins.getInstance();
         if (j!=null) {
```

Comment:
Actually it was an oversight not removing the null check. TaskMethodFinder is only referenced in two places both of which are assured a non-null `theInstance`